tests/testutils.py

```diff
@@ -37,12 +37,19 @@ def setup_test_config(
                 "OPTIONS": {
                     "builtins": [
                         "django_components.templatetags.component_tags",
-                    ]
+                    ],
+                    'loaders': [
```

Comment:
Previously, we loaded component template string from files ourselves with `Path.read_text()`. This allowed us to bypass django's template loaders.

However, now, to support django loaders, we have to load the files through Django.

Which means that also the common Django settings that we use in tests had to be updated to add our template loader `django_components.template_loader.Loader`